packages/be/liftoff.js

```diff
@@ -53,11 +53,11 @@ try {
 // -----------------------------------------------------------------------------
 // const users = async () => {
 //   try {
-//     // const users = await MC.model.User.findOne({ githubUsername: '' })
+//     const user = await MC.model.User.findOne({ githubUsername: 'timelytree' })
```

Comment:
Hardcoded?

Reply:
@orvn this section of `liftoff.js` contains some functions I use to run tests or to modify db entries in bulk when necessary. My username tends to come/go across commits 😅